### DISCUSSION TITLE
Table Head (th) CSS background-color not working

### DISCUSSION THREAD

**Discussion by martinmts** [marp-team/marp#374] [Q&A] [answered]
Hi,

It looks like the `background-color` CSS not working in Marp.
I have the below code:
```
## Test
<style scoped>th {background-color: orange;}</style>

<table width="100%">
    <tr>
        <th>Test Head
        </th>
        <th>Test
        </th>
    </tr>
    <tr>
        <td>- Test
        </td>
        <td>- Test
        </td>
    </tr>
</table>
```
The background-color does not turn to orange at all.

<img width="595" alt="image" src="https://user-images.githubusercontent.com/32575091/202794252-d956797c-72ea-4767-8334-f0502051cbbf.png">

If I assign a class name to the first row, then it will work:
```
## Test
<style scoped>.th {background-color: orange;}</style>

<table width="100%">
    <tr class="th">
        <th>Test Head
        </th>
        <th>Test
        </th>
    </tr>
    <tr>
        <td>- Test
        </td>
        <td>- Test
        </td>
    </tr>
</table>
```
<img width="589" alt="image" src="https://user-images.githubusercontent.com/32575091/202794451-febb1b37-8e0c-4e9d-9c6b-ebcb6118a44f.png">

Did I do something wrong?
Thanks!

**Comment by yhatt** [comment, marked as answer]
Assuming you're using `gaia` theme, try adding  `<thead>` to table header and `<tbody>` to table body explicitly.

```diff
 <table width="100%">
+  <thead>
     <tr>
         <th>Test Head
         </th>
         <th>Test
         </th>
     </tr>
+  </thead>
+  <tbody>
     <tr>
         <td>- Test
         </td>
         <td>- Test
         </td>
     </tr>
+  </tbody>
 </table>
```

`gaia` theme will override the background color of odd rows in the table body through [`tbody > tr:nth-child(odd) th` selector](https://github.com/marp-team/marp-core/blob/bb5d6bbe6c27ba5c25fdfa906099169a9063a295/themes/gaia.scss#L262-L267). (Browser implictly wraps all table rows in `<tbody>` if not defined)

**Reply by martinmts** [reply]
Thank you very much for your solution and explanation! I was indeed using the gaia them, and your suggestion has solved the problem!
